packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -111,6 +112,24 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// position (510.0, 500.0).
   DragStartBehavior dragStartBehavior;
 
+  /// {@template flutter.gestures.monodrag.DragGestureRecognizer.multitouchDragStrategy}
+  /// Configuration the multi-finger drag strategy on the multi-touch devices.
```

Comment:
nit: Maybe this should be `Configure the multi-finger drag strategy on the multi-touch devices.`